agent/tool-scripts/datalog/openvswitch-datalog

```diff
@@ -48,11 +48,20 @@ for b in ${bridges}; do
 	ports_file=${tool_output_dir}/${b}-port-mappings.txt
 	ovs-ofctl show ${b} ${protocols_string} > ${ports_file}
 done
+
```

Comment:
Let's consider merging these two loops since the loop control variable is the same.